detect_secrets_server/storage/core/git.py

```diff
@@ -92,9 +92,19 @@ def get_baseline_file(directory, filename):
 
         # Some repositories may not have baselines.
         # If so, this is a non-breaking error.
-        if not re.match(
-            r"fatal: Path '[^']+' does not exist",
-            error_message,
+        if not (
+            re.match(
+                r"fatal: Path '[^']+' does not exist",
+                error_message,
+            )
+            or
+            # It is possible for the directory you are running detect-secret-server from
+            # to also contain a file with the name as <secrets baseline filename>.
```

Comment:
`# to also contain a file with the name as <secrets baseline filename>.` -> `# to also contain a file with the name as <secrets baseline filename>, but it not be part of the repository.` Perhaps?